packages/api/src/mappings/google/activity.ts

```diff
@@ -32,6 +31,25 @@ export const mapToActivity = (
     metadata: metadata,
   };
 
+  if (googleSessions) {
+    const activitySessions = googleSessions.session.filter(
+      session => session.activityType !== sessionSleepType
+    );
+
+    const activityLogs = activitySessions.map(session => {
+      const isoStringDate = dayjs(Number(session.startTimeMillis)).toISOString();
+
+      return {
+        metadata,
+        name: GoogleActivityTypes[session.activityType],
+        start_time: isoStringDate,
+        end_time: dayjs(Number(session.endTimeMillis)).toISOString(),
```

Comment:
nit: could make a func with `dayjs(Number(millis)).toISOString()` and reuse for `start_time` + `end_time`, also not sure why we're storing `isoStringDate` in a var - doesn't seem needed